yjit/src/codegen.rs

```diff
@@ -10231,7 +10230,6 @@ fn gen_setclassvariable(
     asm.ccall(
         rb_vm_setclassvariable as *const u8,
         vec![
-            Opnd::mem(64, CFP, RUBY_OFFSET_CFP_ISEQ),
```

Comment:
YJIT and ZJIT could theoretically burn in constants here. I wonder why we do CFP offsets